packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -528,12 +535,12 @@ export async function runApiExtraction({
     }
 
     const warningCountAfter = await countApiReportWarnings(projectFolder);
-    if (warningCountAfter > 0 && !ALLOW_WARNINGS.includes(packageDir)) {
+    if (warningCountAfter > 0 && !noBail) {
       throw new Error(
         `The API Report for ${packageDir} is not allowed to have warnings`,
       );
     }
-    if (warningCountAfter === 0 && ALLOW_WARNINGS.includes(packageDir)) {
+    if (warningCountAfter === 0 && allowWarningPkg.includes(packageDir)) {
```

Comment:
should be `noBail` here as well right?

Reply:
mm can be but in that case, when we `allowWarning:true` (allow warnings everywhere ) we will show the messages on every package without errors. Seems to me that this message is to be aware when you allowWarnings on specific packages... wdyt?

Reply:
Ah I see what the difference is now, yeah let's keep as is